deno/lib/index.ts

```diff
@@ -2,3 +2,12 @@ import * as mod from "./external.ts";
 export * from "./external.ts";
 export { mod as z };
 export default mod;
+
+const myArray = mod.array(mod.string()).parse(["foo"], {
+  errorMap: (issue) => {
+    if (issue.code === "too_big") {
+      return { message: "Hey yo, the array must have a length of 2." };
+    }
+    return { message: "Nope" };
+  },
+});
```

Comment:
Does this belong here? It looks possibly left-over?

Reply:
Yeah, this does not belong here.

I don't know why but I think the git hooks enforced by the library have some issues regarding the code gen for `deno`. I don't know, but my PRs always come with some unrelated changes in files that I didn't even touch.